Allow specifying the maximum width of static text in dialogs

schlagert opened the pull request:
This patch allows specification of the maximum width of static texts
created with wxDialog::CreateTextSizer.

The patch preserves backwards compatibility by setting the previous
width value as standard argument.

vadz commented on src/common/dlgcmn.cpp:
```diff
@@ -196,19 +196,21 @@ wxDialogBase::GetParentForModalDialog(wxWindow *parent, long style) const
+                                       wxTextSizerWrapper& wrapper,
+                                       int maxWidth)
 {
+    int widthMax = maxWidth;
```
It's really confusing to have both `maxWidth` and `maxWidth` variables in one and the same function. Could you please either rename the argument to `widthMax` or the local variable to `maxWidth` and just remove the variable declaration/initialization in any case?

vadz commented on interface/wx/dialog.h:
```diff
@@ -292,9 +292,11 @@ class wxDialog : public wxTopLevelWindow
        Splits text up at newlines and places the lines into wxStaticText
-       objects in a vertical wxBoxSizer.
+       objects with the specified maximum width in a vertical wxBoxSizer.
+
```
Please mention that `@a maxWidth` is available since wxWidgets 3.1.0.

schlagert replied:
Sorry, this was done automatically by my default Emacs save-hook.

schlagert commented:
Thanks for the input, I'm going to rebase and amend the PR according to your comments.

schlagert committed:
Allow specifying the maximum width of static text in dialogs

This patch allows specification of the maximum width of static texts
created with wxDialog::CreateTextSizer.

The patch preserves backwards compatibility by setting the previous
width value as standard argument.

schlagert commented:
PR has been amended.

vadz commented:
Merged, thanks!
